src/core/Tile/Tile.test.tsx

```diff
@@ -37,6 +37,7 @@ it('should render new and selected states', () => {
 });
 
 it('should render main text content correctly', () => {
+  window.CSS = { supports: () => true, escape: (i) => i };
```

Comment:
Why this line is needed? For badges?

Reply:
yes. I'm surprised it was not failing before

Reply:
Yeah... Maybe it is worth moving to the global setup?

Reply:
Removed this line and added `supports?.` to Badge